corehq/apps/accounting/tests/test_customer_invoicing.py

```diff
@@ -180,15 +190,9 @@ def test_product_line_items(self):
         invoice = CustomerInvoice.objects.first()
         product_line_items = invoice.lineitem_set.get_products()
         self.assertEqual(product_line_items.count(), 2)
-        for line_item in product_line_items:
-            self.assertIn(line_item.base_description, [
-                'One month of CommCare Advanced Edition Software Plan.',
-                'One month of CommCare Standard Edition Software Plan.',
-            ])
-            self.assertIn(line_item.base_cost, [
```

Comment:
Looks like test coverage is being removed re: ```line_item.base_cost```.  Is that ok/desirable?

Reply:
Nope. That's definitely a mistake